Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -169,6 +172,19 @@ CSSStyleSheet* Parser::parse_as_css_stylesheet(Optional<URL::URL> location)
             log_parse_error();
             continue;
         }
+        // https://drafts.csswg.org/css-cascade-5/#at-import
+        // Any @import rules must precede all other valid at-rules and style rules in a style sheet (ignoring @charset and @layer statement rules)
+        // and must not have any other valid at-rules or style rules between it and previous @import rules, or else the @import rule is invalid.
+        auto is_layer_statement_rule = is<CSSLayerStatementRule>(*rule);
+        auto is_import_rule = is<CSSImportRule>(*rule);
+
+        // FIXME: Include @charset rules in this check when they are added.
```

Comment:
This FIXME isn't necessary. It's confusing, but `@charset` is a weird beast that isn't actually an at-rule that shows up in the CSSOM. https://drafts.csswg.org/css-syntax/#charset-rule

Reply:
...there are order-requirements for `@namespace` though, and I think some others.